CHANGELOG.next.asciidoc

```diff
@@ -103,6 +103,9 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Rename bad ECS field name tracing.trace.id to trace.id in aws elb fileset. {pull}22571[22571]
 - Fix parsing issues with nested JSON payloads in Elasticsearch audit log fileset. {pull}22975[22975]
 - Rename `network.direction` values in crowdstrike/falcon to `ingress`/`egress`. {pull}23041[23041]
+- Add User Agent Parser for Azure Sign In Logs Ingest Pipeline {pull}23201[23201]
```

Comment:
These lines should have been added accidentally.

Reply:
Yes indeed, corrected in https://github.com/elastic/beats/pull/25368/commits/ceeee698514d1293ef0317f1f452dcc004ccc762